addons/website_crm_partner_assign/models/crm_lead.py

```diff
@@ -190,17 +190,15 @@ def search_geo_partner(self):
 
     @api.multi
     def partner_interested(self, comment=False):
-        self.check_access_rights('write')
         message = _('<p>I am interested by this lead.</p>')
         if comment:
             message += '<p>%s</p>' % comment
         for lead in self:
             lead.message_post(body=message, subtype="mail.mt_note")
-            lead.sudo().convert_opportunity(lead.partner_id.id)
+            lead.sudo().convert_opportunity(lead.partner_id.id)  # sudo required to convert partner data
```

Comment:
Good catch ! Seems that it is required only for the `write` since the `write` needs the `read` to wins against the access rights. Or something like that !